src/makeCalculator.js

```diff
@@ -4,7 +4,53 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    add(value) {
+      result += value;
+
+      return this;
+    },
+
+    subtract(value) {
+      result -= value;
+
+      return this;
+    },
+
+    multiply(value) {
+      result *= value;
+
+      return this;
+    },
+
+    divide(value) {
+      if (value !== 0) {
+        result /= value;
+
+        return this;
+      }
```

Comment:
Remember that the `operate` method should be able to be called in a chain. Currently, if `divide` is called with a value of 0, it does not return `this`, which breaks the chaining. You should ensure that `this` is always returned regardless of the input value. Consider handling the division by zero case separately while still allowing for method chaining. [CODE KNOWLEDGE]